### PR TITLE
feature1

### PR DESCRIPTION
**Pull request by johnmcg04**
feature1

### CODE REVIEW

**Comment by UserConor** [comment]
Looks great!

**Review by UserConor — APPROVED** [review]

File: feature1.sql [hunk 1]
```diff
@@ -0,0 +1,16 @@
+use KainoosProjectTeam4_JohnMc;
```
Comment (jkelso662) [inline feature1.sql]: Would love USE to be caps!

File: feature1.sql [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+CREATE TABLE Employee(
+	employeeId INT AUTO_INCREMENT PRIMARY KEY,
+	firstName VARCHAR(80),
```
Comment (jkelso662) [inline feature1.sql:5]: Indentation here is diff to the rest of the file!

**Review by jkelso662 — CHANGES_REQUESTED** [review]
Don't see any employees yet. Feature is no use if there aren't any employees in the db!

**Comment by jkelso662** [comment]
Would also need to see a descriptive merge request description to describe what these changes represent/the task it relates to. Also, for future reference `feature1` ain't the best branch name. Customer has just asked me, "What does feature1 mean?"!

**Review by Breanna-loughrey — APPROVED** [review]

**Comment by Breanna-loughrey** [comment]
yes

File: feature1.sql [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+	surname VARCHAR(80),
+	businessSector VARCHAR(30)
+);
+
+CREATE TABLE Sales_Employees(
+	employeeId INT,
+	commission_Rate Decimal(5,2),
+    CONSTRAINT chkCommissionRate CHECK (commission_Rate >= 0 AND commission_Rate <= 100),
```
Comment (jkelso662) [inline feature1.sql:13]: Indentation issue! But I won't block the merge.

**Review by jkelso662 — APPROVED** [review]
Much better! Will definately want to see a good merge request description next time. And watch out for simple things like CAPS and indentation. :)

**Comment by lullabelle** [comment]
Hey can you put your team name in the ReadMe?

**Pull request merged**